current_fall2025/scripts/app.py

```
@@ -229,41 +307,54 @@ def main():
     # Logic Loop
     if st.session_state.messages and st.session_state.messages[-1]["role"] == "user":
         query_text = st.session_state.messages[-1]["content"]
-        
+
+        # Check if we're retrying after an error
+        is_retry = (
```

Ruff (0.14.8)

Local variable `is_retry` is assigned to but never used

Remove assignment to unused variable `is_retry`

(F841)